### ISSUE TITLE
[UL2] How can I reproduce the in-context few-shot results?

### ISSUE THREAD

**Issue by danny980521** [google-research/google-research#1426]
Hi, I would like to reproduce the results listed in table10 and table11 of the UL2 paper.
I did some experiments with UL2 20B and most of resulted scores were lower than the paper's. ex) `SuperGLUE-copa` & `rte`, `XSUM`  
So I have some questions with respect to reproducing your results. (or nearly the same)

1. Should I use the appropriate instruction when configuring the prompt? 
    ex) `"After reading this passage, I have a question: {question}? True or False?"` for SuperGLUE-boolq
2. Did you use the sentinel token when running few-shot experiment just like one in a pre-trained manner?
3. Should I do any post-processing like normalization? 
    ex) As far as I know, in the GPT-3 paper, authors normalized the unconditional probability of each completion, by computing `{P(completion|context)/P(completion|answer context)}`.
4. For SuperGLUE tasks, did you use perplexity to decide the best option?
5. For XSUM task, what decoding strategy did you use? 
    ex) simple greedy decoding, beam search, ... 

Thanks.

**Comment by danny980521** [comment]
To add one more question, when I did a few-shot ICL experiment with T5.1.1-XXL and UL2 20B, the performance was generally better in one-shot than in zero-shot. 
However, the performance tended to decrease from a certain number of shots, and the number of shots at which this performance began to decrease was smaller in UL2 than in T5. 

I wonder if the authors also experienced this trend.

Thank you.

cc: @MostafaDehghani
